### DISCUSSION TITLE
Support for geometry operators

### DISCUSSION THREAD

**Discussion by manuganji** [PostgREST/postgrest#2035] [Q&A] [answered]
Hi, is it possible to use geometry functions for querying. The list is here:

https://www.postgresql.org/docs/current/functions-geometry.html#FUNCTIONS-GEOMETRY-FUNC-TABLE

**Comment by wolfgangwalther** [comment, marked as answer]
Please clarify: Are you looking to use operators on geometric types as filter (i.e. in the `WHERE` clause) or are you looking to use some of those functions in the `SELECT` part?

**Reply by wolfgangwalther** [reply]
Created an issue for this as it's more of a feature request than a discussion.
